activerecord-crate-adapter.gemspec

```diff
@@ -46,9 +46,11 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec", "~> 2.14"
+  spec.add_development_dependency "rspec", "~> 3.13"
 
   spec.add_dependency('activerecord', '~> 4.1.0')
   spec.add_dependency('arel', '>= 5.0.0')
   spec.add_dependency('crate_ruby', '~> 0.2.0')
+  # https://github.com/ruby/bigdecimal#which-version-should-you-select
+  spec.add_dependency('bigdecimal', '~> 1.4')
```

Comment:
The usage of `BigDecimal.new` within `activerecord` itself currently blocks further Ruby version upgrades. As per that link above, Ruby 2.7 is the latest version still supporting `BigDecimal.new`.